Sitemap/Voter.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Symfony\Cmf\Bundle\SeoBundle\Sitemap;
+
+/**
+ * A chain as a decorator for all tagged voter services.
+ *
+ * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
+ */
+class Voter extends AbstractChain implements VoterInterface
+{
+    /**
+     * {@inheritDocs}
+     *
+     * First comes first - The first negative voter decides.
```

Comment:
in terms of symfony security, this would be called "unanimous": every voter can veto a content.